lib/top/mdx_top.mli

```diff
@@ -22,11 +22,17 @@ open Result
 type t
 (** The type for configuration values. *)
 
+type directive =
+  | Directory of string
+  | Load of string  (** The type for toplevel directives *)
+
+val directories : string list -> directive list
```

Comment:
This function is not used anywhere

Reply:
In fact it is: in the generated executable. I added it to the generated executable.